configs/webdriver.json

```diff
@@ -2,24 +2,20 @@
   "index_name": "webdriver",
   "start_urls": [
     {
-      "url": "http://webdriver.io/guide.html",
+      "url": "http://beta.webdriver.io/docs/gettingstarted.html",
```

Comment:
It seems you modified webdriver.json here too @s-pace

Reply:
Yep under control, I will discard these changes